examples/ledger_nano_transaction.rs

```diff
@@ -3,6 +3,9 @@
 
 //! cargo run --example ledger_nano_transaction --features=ledger_nano --release
 
+use std::env;
+
+use dotenv::dotenv;
```

Comment:
Can you not import them and use the scopes like in the other examples? Since this is shown in the wiki I think it's better to focus the imports only on the client

Reply:
I copied it from another example like this :trollface:

Reply:
Fair enough, I didn't clean all examples yet :(